.circleci/config.yml

```diff
@@ -379,6 +379,14 @@ jobs:
     environment:
       TEST_SUITE: src/__tests__/graphql-codegen-ios.test.ts
       CLI_REGION: us-west-1
+  build-app-ts-e2e-test:
```

Comment:
These changes to config should go into `.circleci/config.base.yml` and I believe this file gets overridden by config generated by running `yarn split-e2e-tests`.